main.go

```diff
@@ -447,45 +425,44 @@ func runWatcher(fileChangeCommand string, dirs []string, killServer bool) error
 		for {
 			select {
 			case event := <-w.Event:
-				Debug.log("Full path is:  " + event.Path)
+				ControllerDebug.log("Watch path is:  " + event.Path)
```

Comment:
Is this the start of a file change trigger? If so, let's add that to the debug message. Something like "File change detected at path /.../.../..."